buf/internal/toolchain.bzl

```diff
@@ -16,122 +16,83 @@
 
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "update_attrs")
 
-_TOOLCHAINS_REPO = "rules_buf_toolchains"
-
-_BUILD_FILE = """
-load(":toolchain.bzl", "declare_buf_toolchains")
+_TOOLS = ["buf", "protoc-gen-buf-breaking", "protoc-gen-buf-lint"]
 
-package(default_visibility = ["//visibility:public"])
+_TOOLCHAINS_REPO = "rules_buf_toolchains"
 
-declare_buf_toolchains(
-    os = "{os}",
-    cpu = "{cpu}",
-    rules_buf_repo_name = "{rules_buf_repo_name}",
- )
-"""
+BufToolchainInfo = provider(fields = {
+    "buf_tool": "A File for a buf executable",
+    "protoc_breaking_tool": "A File for protoc-gen-buf-breaking executable",
```

Comment:
I think it's best to name them `protoc_gen_breaking_tool` and `protoc_gen_lint_tool`

Reply:
will change